src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -45,6 +44,13 @@ public static void GetDirectoryName(string path, string expected)
             Assert.Equal(expected, Path.GetDirectoryName(path));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".Net Core returns null dotnet/corefx #19337")]
+        public static void GetDirectoryName_EmptyString_Throws()
+        {
+            Assert.Throws<ArgumentException>(() => Path.GetDirectoryName(string.Empty));
+        }
```

Comment:
How are we tracking / differentiating Skips like this that we plan to fix vs ones where we've accepted the behavior and there's no further action?  Is there a rush to suppress the desktop failure this morning vs fixing the issue in core and then fixing it this evening or whenever the new build is available?